Api/CmsEntityConverterInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Overdose\CMSContent\Api;
+
+interface CmsEntityConverterInterface
+{
+    const CMS_MEDIA_NODE = 'media';
+    const PAGE_ENTITY_CODE = 'pages';
+    const BLOCK_ENTITY_CODE = 'blocks';
+
+    /**
+     * @return string
```

Comment:
Need to describe what this function must do ( correct description must be everywhere )